### DISCUSSION TITLE
The Time Zone

### DISCUSSION THREAD

**Discussion by palicepta** [Status-Page/Status-Page#492] [Ideas]
Dear @HerrTxbias,

Could you please add the Time zone to Create/Update date?

Now:
![image](https://user-images.githubusercontent.com/112872766/225332074-1ff753c1-ee13-4ae3-90ae-0b6dd198e180.png)

Expected something like this:
![image](https://user-images.githubusercontent.com/112872766/225332619-6449c25d-0a3a-4e1f-aa43-bdfb2a2e4945.png)

Or if it's possible, take info from customer's Time zone and set time from it.

Thank you in advance!

**Comment by herrtxbias** [comment]
You should be able to include the Timezone yourself in your configuration.py:
https://github.com/Status-Page/Status-Page/blob/main/statuspage/statuspage/configuration_example.py#L161-L168
